### PR TITLE
perf test for 64-bit intrin with beachmark.js

### PR DESCRIPTION
**Pull request by lionkunonly**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [ ] I agree to contribute to the project under OpenCV (BSD) License.
- [ ] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [ ] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by lionkunonly** [comment]
@huningxin . I am close to finish the perf tests for 64-bit intrinsics. But in this afternoon, I find my chrome can not access the online js files which is necessary  like `lodash.js`, `beachmark.js`, `platform.js` from [https://cdnjs.cloudflare.com](https://cdnjs.cloudflare.com). I think it may  be caused by the proxy. I have asked others who use the same proxy as mine to try to access these files, and they failed, too.

So I am not able to test it now, but I think it is close to the final version. I will verify it when my proxy is Ok. Please review it and give me suggestions, thanks.

**Comment by lionkunonly** [comment]
These perf tests are verified successfully. Please review it.

File: modules/js/perf/perf_64bits.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    } else {
+      log("no getting invalid params, run all the cases with Mat of shape (1000 x 1000)");
+      sizeArray = [1000, 1000];
+    }
+    addCountNonZeroCase(suite);
+    addMatDotCase(suite);
+    addSplitCase(suite);
+    addMergeCase(suite);
+    setInitParams(suite, sizeArray)
+    setBenchmarkSuite(suite);
```
Comment (huningxin) [inline modules/js/perf/perf_64bits.js:144]: Can it log out the performance data for each kernel? Could you please share an example result? Thanks!

Comment (lionkunonly) [reply modules/js/perf/perf_64bits.js:144]: Of course. Please see the example screenshot.

PerfExample

Comment (huningxin) [reply modules/js/perf/perf_64bits.js:144]: Great!

**Review by huningxin — APPROVED** [review]
Looks good to me.

**Comment by lionkunonly** [comment]
#340

**Comment by huningxin** [comment]
As @terfendail is out of office, I am going to merge this PR. @lionkunonly , after this PR merged, please collect the initial data and post to #340.

**Pull request merged**